public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -276,4 +277,20 @@ else if ((stratum1 != null && !stratum1.isEmpty())) {
         resp.setItems(resultList.stream().map(TO_CLIENT_DBDOMAIN).collect(Collectors.toList()));
         return ResponseEntity.ok(resp);
     }
+
+    @Override
+    public ResponseEntity<DbDomainListResponse> getDomainFilters() {
+        List<DbDomain> domains=dbDomainDao.findByDbType("domain_filter");
```

Comment:
Looks like you don't use that other dao method anyway.

Reply:
I changed the dao to use concept id too.